auction-app-frontend/src/layout/navbar/Navbar.jsx

```diff
@@ -51,27 +52,29 @@ const Navbar = () => {
       <div className='navbar__white'>
         <div className='navbar__container--white'>
           <Logo />
-          <div className='navbar__container--search_and-links'>
-            <SearchField
-              searchTerm={searchTerm}
-              onSearchTermChange={onSearchTermChange}
-              categoryId={activeCategory}
-              onSearchIconClick={onSearchIconClick}
-              setSearchTerm={setSearchTerm}
-              setSearchProducts={setSearchProducts}
-              setProducts={setProducts}
-            />
-            <div className='navbar__navigation'>
-              {navlinks.map((link) => (
-                <NavbarLink
-                  link={link}
-                  key={link.key}
-                  activeLink={activeLink}
-                  onClick={setActiveLink}
-                />
-              ))}
+          {!(pathname.includes('login') || pathname.includes('register')) ? (
```

Comment:
Instead of `condition ? true : false` you can use `condition && true`.

https://reactpatterns.js.org/docs/logical-and-operator/